example/http/server/server.go

```diff
@@ -62,6 +62,7 @@ func main() {
 			trace.WithAttributes(attrs...),
 			trace.ChildOf(spanCtx),
 		)
+		span.SetAttributes(entries...)
```

Comment:
Shouldn't we instead have the SDK apply these, internally? I.e., I would expect to see the dctx entries included in the span as first-class distributed correlations, not as span attributes.

Reply:
I agree. It would be nice to have the dctx entires in the SDK so that application does not need to set them explicitly. However, I don't see there is a `distributed correlations` in the span struct. Could you provide any guidance (I am happy to try fixing this)? Thanks.